src/services/workspace-client/index.ts

```diff
@@ -85,15 +79,17 @@ export abstract class WorkspaceClient {
     if (keycloak) {
       return new Promise((resolve, reject) => {
         keycloak.updateToken(minValidity).success((refreshed: boolean) => {
-          if (refreshed && keycloak.token) {
-            const header = 'Authorization';
-            this.axios.defaults.headers.common[header] = `Bearer ${keycloak.token}`;
+          console.log('debug: refreshToken, minValidity', minValidity, ', config', config, ', refreshed', refreshed);
```

Comment:
it should be removed before merging, right?

Reply:
yes, @artaleks9 hasn't reproduced the issue so I'll get rid of these outputs.